### DISCUSSION TITLE
Add possibility to merge object together

### DISCUSSION THREAD

**Discussion by jordivn** [dani-garcia/vaultwarden#4945] [Ideas]
Option to check the vault list for double items and let them merge together (aka like contacts of google).

Mobile creates an diffrent account for apps. Would be nice if we can clean the list in this way

**Comment by BlackDex** [comment]
That would be a client feature and this project does not maintain the clients.

**Reply by jordivn** [reply]
Do you have an link to the proper repo for me?

**Reply by BlackDex** [reply]
https://github.com/bitwarden/clients
